docker/Makefile

```diff
@@ -1,4 +1,5 @@
 PUPPERWARE_ANALYTICS_STREAM ?= dev
+NAMESPACE ?= puppet
```

Comment:
Should make this change on Windows side as well

Reply:
This would never be used in CI, only by devs while they build locally with their changes.

What would we change on the Windows side for this (if that even is desirable)?

Reply:
If someone is dev'ing on non-Windows, they should be able to dev on Windows as well

Reply:
Sure, but that's what I was asking you about.

The `build.ps1` file has a `Namespace` parameter for the `Build-Container` function, which seems like enough.

But that's not used the same way the Makefile is, so there's no corresponding change on the Windows side I don't think?